models/weather-fetcher.js

```diff
@@ -0,0 +1,17 @@
+var rest = require('restler');
+var parser = require('xml2json');
+
+module.exports.fetch = function() {
+    
+    return new Promise(function(resolve, reject) {
+        var result = {};
+        rest.get('http://www.kma.go.kr/wid/queryDFSRSS.jsp?zone=2723067100')
```

Comment:
I think the zone code(2723067100 in this case) should be passed as a parameter of this funtion.